pyEpiabm/pyEpiabm/sweep/host_progression_sweep.py

```diff
@@ -29,6 +29,10 @@ def __init__(self):
         is also initialised and associated parameters are called from the
         parameters class.
 
+        Infectiousness progression defines an array to scale a person's
+        infectiousness that depends on time since the start of the infection,
+        measured in timesteps (following what is done in Covidsim).
```

Comment:
```suggestion
        Infectiousness progression defines an array used to scale a person's
        infectiousness and which depends on time since the start of the
        infection, measured in timesteps (following what is done in Covidsim).
```